test/functional/main_test.go

```diff
@@ -98,11 +98,12 @@ const (
 
 	// resources and misc functionality.
 
-	featureScratch = "Scratch" // validate scratch layer mounting
-	featurePlan9   = "Plan9"   // Plan9 file shares
-	featureSCSI    = "SCSI"    // SCSI disk (virtuall and physical) mounts
-	featureVSMB    = "vSMB"    // virtual SMB file shares
-	featureVPMEM   = "vPMEM"   // virtual PMEM mounts
+	featureScratch  = "Scratch"  // validate scratch layer mounting
+	featurePlan9    = "Plan9"    // Plan9 file shares
+	featureSCSI     = "SCSI"     // SCSI disk (virtuall and physical) mounts
+	featureVSMB     = "vSMB"     // virtual SMB file shares
+	featureVPMEM    = "vPMEM"    // virtual PMEM mounts
+	featureHVSocket = "HVSocket" // Hyper-V socket functionality
```

Comment:
Are there any scenarios/configurations in which HVsocket won't be supported and these tests can't be run?

Reply:
i'm fairly certain hyper-v sockets are a requirement for HCS, and i don't think anything needs to be done to enable it
documentation says anything post windows 10/server 2016 supports it: https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/make-integration-service

Reply:
In that case do we need this feature flag? We either run all tests and run these with them or we specify regex that excludes these tests, right?